### PR TITLE
removed all template files etc.

### PR DESCRIPTION
**Pull request by miniJmo**

### CODE REVIEW

**Comment by miniJmo** [comment]
so... I added a little more than first intended

**Review by Taoshix — APPROVED** [review]
こんいちは、Miniさん、それらの変化はとても面白いね。頑張れ！

File: CentralHub.WebUI/Pages/Trackers.razor [hunk 1]
```diff
@@ -21,6 +21,7 @@ else
         <thead>
         <tr>
             <th>Name</th>
+            <th> Location</th>
```
Comment (mads256h) [inline CentralHub.WebUI/Pages/Trackers.razor:24]:
```suggestion
            <th>Location</th>
```

File: CentralHub.Api.Model/Tracker.cs [hunk 2]
```diff
@@ -2,9 +2,10 @@ namespace CentralHub.Api.Model;
 
 public class Tracker
 {
-    public Tracker(string name, string macAddress, TrackerType trackerType)
+    public Tracker(string name, string location, string macAddress, TrackerType trackerType)
     {
         Name = name;
+        Location = location;
```
Comment (mads256h) [inline CentralHub.Api.Model/Tracker.cs:8]: We should probably add a Room class

**Review by mads256h — CHANGES_REQUESTED** [review]

File: CentralHub.Api.Model/Tracker.cs (hunk 2 above)
Comment (miniJmo) [reply CentralHub.Api.Model/Tracker.cs:8]: yeah det er nok en god ide

**Review by mads256h — APPROVED** [review]

**Pull request merged**